cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -68,6 +68,20 @@ if(OMR_HOST_ARCH STREQUAL "s390")
 	list(APPEND OMR_PLATFORM_COMPILE_OPTIONS -march=z9-109)
 endif()
 
+if(OMR_ARCH_AARCH64 AND (CMAKE_C_COMPILER_VERSION VERSION_GREATER_EQUAL 9))
```

Comment:
`VERSION_GREATER_EQUAL` is not supported by the minimum version cmake; this needs to be:
```
if(OMR_ARCH_AARCH64 AND NOT (CMAKE_C_COMPILER_VERSION VERSION_LESS 9))
```
See also https://github.com/eclipse-openj9/openj9/blob/master/runtime/cmake/platform/toolcfg/gnu.cmake#L24.